src/kzg.rs

```diff
@@ -0,0 +1,10 @@
+use crate::primitives::Bytes32;
+use crate::ssz::ByteVector;
+use ssz_rs::prelude::*;
+
+const KZG_COMMITMENT_BYTES_LEN: usize = 48;
+const KZG_PROOF_BYTES_LEN: usize = 48;
```

Comment:
a bit random but may be worth making these `pub`, in the event some user of the library needed them for some reason...

Reply:
Done. Make sense 👍